Added tests, default image and changed default namespace (#1)
appuio/component-openshift4-keepalived: docs/modules/ROOT/pages/references/parameters.adoc

The channel from where the operator should be fetched.

== `openshift_version`

[horizontal]
type:: string
default:: `4.9`

> The openshift version which will be used as the image tag in KeepalivedGroup CRD.

Reviewer: This is not entirely true, since it gets appended to `v`.

```suggestion
The OpenShift version used to set the corresesponding image tag for the KeepalivedGroup(s).
```

Author: yep, missed that, comited your suggestion
